### PR TITLE
[CreateApp] make shortcuts work from windows

### PR DESCRIPTION
**Pull request by thoys**
Addresses the following tickets: 
- https://highfidelity.manuscript.com/f/cases/19063/Shortcuts-work-when-UI-has-focus
- https://highfidelity.manuscript.com/f/cases/15026/Ctrl-P-Parent-takes-a-screenshot

Test plan is available in ticket:
- https://highfidelity.manuscript.com/f/cases/19063/Shortcuts-work-when-UI-has-focus

### CODE REVIEW

File: interface/resources/qml/hifi/tablet/EditEntityList.qml [hunk 1]
```diff
@@ -10,6 +10,7 @@ import "../../styles-uit"
 
 WebView {
     id: entityListToolWebView
-    url: Paths.defaultScripts + "/system/html/entityList.html"
+    url: Paths.defaultScripts + "/system/html/entityListf.html"
```
Comment (thoys) [inline interface/resources/qml/hifi/tablet/EditEntityList.qml]: Oops, left this accessibility test here.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14673/HighFidelity-Beta-PR14337-d98ac09.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14673/HighFidelity-Beta-PR14337-d98ac09.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14673/HighFidelity-Beta-Interface-PR14337-d98ac09.dmg

File: scripts/system/html/js/gridControls.js [hunk 2]
```diff
@@ -131,15 +129,24 @@ function loaded() {
...
-              EventBridge.emitWebEvent(JSON.stringify({ type: 'parent' }));
-          }
-      }
-    })
+
+    document.addEventListener("keyup", function (keyUpEvent) {
+        if (keyUpEvent.target.nodeName === "INPUT") {
+            return;
+        }
+        let {code, key, altKey, ctrlKey, shiftKey} = keyUpEvent;
```
Comment (huffman) [inline scripts/system/html/js/gridControls.js]: fancy

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14916/HighFidelity-Beta-PR14337-d98ac09.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14916/HighFidelity-Beta-PR14337-d98ac09.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14916/HighFidelity-Beta-Interface-PR14337-d98ac09.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14916/HighFidelity-Beta-PR14337-d98ac09.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14916/HighFidelity-Beta-Interface-PR14337-d98ac09.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15091/HighFidelity-Beta-PR14337-d98ac09.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15091/HighFidelity-Beta-PR14337-d98ac09.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15091/HighFidelity-Beta-Interface-PR14337-d98ac09.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15091/HighFidelity-Beta-PR14337-d98ac09.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15091/HighFidelity-Beta-Interface-PR14337-d98ac09.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15114/HighFidelity-Beta-PR14337-f5d933a.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15119/HighFidelity-Beta-PR14337-2c715bf.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15119/HighFidelity-Beta-PR14337-2c715bf.dmg

**Comment by thoys** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15137/HighFidelity-Beta-PR14337-2c715bf.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15137/HighFidelity-Beta-PR14337-2c715bf.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15137/HighFidelity-Beta-Interface-PR14337-2c715bf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15137/HighFidelity-Beta-PR14337-2c715bf.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15137/HighFidelity-Beta-Interface-PR14337-2c715bf.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Fail @conklin94122 

Talked to the dev about Mac keybindings.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15148/HighFidelity-Beta-PR14337-cd12dd8.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15148/HighFidelity-Beta-PR14337-cd12dd8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15148/HighFidelity-Beta-Interface-PR14337-cd12dd8.exe

**Comment by thoys** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15153/HighFidelity-Beta-PR14337-cd12dd8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15153/HighFidelity-Beta-PR14337-cd12dd8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15153/HighFidelity-Beta-Interface-PR14337-cd12dd8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15153/HighFidelity-Beta-PR14337-cd12dd8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15153/HighFidelity-Beta-Interface-PR14337-cd12dd8.exe

**Comment by QliCknox** [comment]
Pass @conklin94122 

Shortcuts work with create windows focused

**Pull request merged**